roles/install/vars/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+__default_pkgs:
+  - lemonldap-ng
+  - lemonldap-ng-doc
+
+__redhat_apache_pkgs:
+  - httpd
+  - mod_perl
+  - mod_fcgid
+__redhat_nginx_pkgs:
+  - nginx
+  - lemonldap-ng-fastcgi-server
+  - lemonldap-ng-nginx
```

Comment:
Starting with llng 2.19.0, -nginx subpackage doesn't exist anymore, the nginx configurations have been move to their respective sub-packages.